### PR TITLE
replacement of c3-angular

### PR DESCRIPTION
**Pull request by kamleshverma1**
added angular patternfly instead of c3-angular

Signed-off-by: kamlesh kaverma@gmail.com

### CODE REVIEW

File: app/components/dashboard/dashboard-controller.ts [hunk 1]
```diff
@@ -77,7 +87,7 @@ export class DashboardController {
          this.totalCapacity = {
                 freeGB: 0, usedGB: 0, totalGB: 0,
                 freeFormatted: '0 B', usedFormatted: '0 B', totalFormatted: '0 B'
-            };
+         };
```
Comment (kmkanagaraj) [inline app/components/dashboard/dashboard-controller.ts]: The code used for the older charting should be removed

File: app/components/dashboard/dashboard-controller.ts [hunk 2]
```diff
@@ -96,6 +106,42 @@ export class DashboardController {
...
+         this.chartConfig = {
+             'chartId': 'donutChart',
+             'units': 'TB',
+             "legend":{"show":true},
+             'tooltipFn': function (d) {
+               return '<span class="donut-tooltip-pf"style="white-space: nowrap;">' +
+                        d[0].value + ' ' + d[0].name +
+                      '</span>';
+               }
```
Comment (kmkanagaraj) [inline app/components/dashboard/dashboard-controller.ts]: is it possible to have a handler for click event?

File: bower.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "name": "usm-client",
   "version": "0.0.1",
   "dependencies": {
-    "restangular": "~1.5.1",
-    "c3-angular": "~0.6.0"
+    "angular-patternfly": "patternfly/angular-patternfly#~2.1.1",
```
Comment (kmkanagaraj) [inline bower.json]: can this be?

"angular-patternfly": "~2.2.0"

File: app/components/dashboard/dashboard-controller.ts (hunk 1 above)
Comment (kamleshverma1) [inline app/components/dashboard/dashboard-controller.ts]: ya , we can remove initial value for totalCapacity . But it's being used . and using this only i am getting usedSize, freeSize , we can't remove this variable completely.

**Pull request closed without merging**